src/module.c

```diff
@@ -186,11 +186,14 @@ int TSDB_add(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     RedisModuleKey *key = RedisModule_OpenKey(ctx, keyName, REDISMODULE_READ|REDISMODULE_WRITE);
 
     double timestamp, value;
+    RedisModuleString* automaticTimestamp = RedisModule_CreateStringPrintf(ctx, "*");
```

Comment:
@yaeltzirulnikov using RedisModule_CreateStringPrintf is not very optimal, please check the RedisModule's API to see if you can compare strings without allocating every time `TSDB_add` is called.